snuba/pipeline/stages/query_processing.py

```diff
@@ -0,0 +1,45 @@
+from typing import Any, cast
+
+from snuba.clickhouse.query import Query as ClickhouseQuery
+from snuba.datasets.entities.factory import get_entity
+from snuba.datasets.plans.query_plan import QueryRunner
+from snuba.pipeline.composite import CompositeExecutionPipeline
+from snuba.pipeline.query_pipeline import QueryPipelineData, QueryPipelineStage
+from snuba.query.composite import CompositeQuery
+from snuba.query.data_source.simple import Table
+from snuba.query.logical import Query as LogicalQuery
+from snuba.request import Request
+
+
+class EntityAndStoragePipelineStage(
```

Comment:
I know we eventually want to break this stage up into separate entity processing and storage processing stages. But could write a comment describing what this stage is responsible for now (e.g. execute entity logical processors, apply query translator, apply translation mappers, storage processors, etc.)?